test/helpers/types/otlp/types.go

```diff
@@ -37,21 +37,32 @@ type LogRecord struct {
 	TimeUnixNano         string      `json:"timeUnixNano,omitempty"`
 	ObservedTimeUnixNano string      `json:"observedTimeUnixNano,omitempty"`
 	SeverityNumber       int         `json:"severityNumber,omitempty"`
+	SeverityText         string      `json:"severityText,omitempty"`
 	TraceID              string      `json:"traceId,omitempty"`
 	SpanID               string      `json:"spanId,omitempty"`
 	Body                 StringValue `json:"body,omitempty"`
 	Attributes           []Attribute `json:"attributes,omitempty"`
 }
 
 type AttributeValue struct {
-	StringValue string      `json:"stringValue,omitempty"`
+	StringValue *string     `json:"stringValue,omitempty"`
 	Bool        bool        `json:"boolValue,omitempty"`
-	Int         int         `json:"intValue,omitempty"`
+	Int         *string     `json:"intValue,omitempty"` //proto example defines as a string
```

Comment:
The comment certainly raised an eyebrow, but the example is actually correct (from the spec):

> Note that according to [Protobuf specs](https://developers.google.com/protocol-buffers/docs/proto3#json) 64-bit integer numbers in JSON-encoded payloads are encoded as decimal strings, and either numbers or strings are accepted when decoding.

My guess is that this is to get around languages not having any native integer type (JavaScript)...